common/lib/common-definitions/BREAKING.md

```diff
@@ -0,0 +1,7 @@
+## 0.21 Breaking changes
+
+- [supportsTags deprecated](#supportstags-deprecated)
+
+### supportsTags deprecated
+We are deprecating use of supportsTags and instead going to handle tags directly at the interface of the runtime and
+`IContainerContext`. This will allow loggers downstream to be agnostic about whether messages have tags. This will involve a change in `container-definitions`, which is packaged as part of the `build-common` release.
```

Comment:
```suggestion
We are deprecating use of supportsTags and instead going to handle tags directly at the loader/runtime boundary
via `IContainerContext`. Other than this boundary, all loggers can assume full support for tags on all telemetry events. 
This will involve a change in `container-definitions`, which is packaged as part of the `build-common` release.
```